index.md

```diff
@@ -1,43 +1,54 @@
 
-## Current Status: ## 
-The earliest releases of rdiff-backup are more than seven years old. Since then there have been more than 70 releases fixing bugs and adding features. The basic functionality on unix platforms has been tested by many people over this time and can be considered stable.
+The wiki contains documentation on use cases, and serves as a respository of scripts for people to contribute to. These scripts do many things for and around rdiff-backup.
 
-Many users seem to use rdiff-backup on MS Windows but this configuration is less well tested. Also, features such as Mac OS X resource forks, Extended Attributes, and Access Control Lists were only released about five years ago. There are no known bugs in these newer features, but they are not as thoroughly tested as the basic functionality. Native Windows support was first released six months ago.
+## Authors and Contributors ## 
 
-Using rdiff-backup to backup files to a server mounted via smbfs or CIFS has been a troublesome configuration for some users. Mounting via smbfs tends to be more reliable than CIFS, although it is deprecated on Linux and does not support files greater than 2 GB. See [the FAQ](FAQ.html) for more on this setup.
+[@sol1][sol1gh] is the current maintainer of rdiff-backup
+
+Project Lead / Maintainer History:
+
+* Edward Ned Harvey, maintainer 2012 to 2016 
+* Andrew Ferguson, maintainer 2008 to 2012
+* Dean Gaudet, maintainer 2006 to 2007
+* Ben Escoto, original author, maintainer 2001 to 2005
+
+Other code contributors are:
+
+* Daniel Hazelbaker, who contributed Mac OS X resource fork support. (July 2003)
+* Dean Gaudet, for checking in many patches, and for finding and fixing many bugs.
+* Andrew Ferguson, for improving Mac OS X support and fixing many user-reported bugs.
+* Josh Nisly, for contributing native Windows support. (June 2008)
+* Fred Gansevles, for contributing Windows ACLs support. (July 2008)
+
+[sol1gh]: https://github.com/sol1
+
+Thanks also to:
+
+* The [Free Software Foundation][fsf] for previously hosting rdiff-backup with their [Savannah][sav] system.
+* Andrew Tridgell and Martin Pool for writing rdiff, and also for rsync, which gave me the idea
+* Martin Pool and Donovan Baarda for their work on librsync, which rdiff-backup needs.
+* Michael Friedlander for initially acting interested in the idea
+        and giving me accounts for testing
+* Lots of people on the mailing list for their helpful comments,
+        advice, and patches, particularly Alberto Accomazzi, Donovan Baarda,
+* Jeb Campbell, Greg Freemyer, Jamie Heilman, Marc Dyksterhouse, and Ralph Lehmann.
+
+[fsf]: http://www.fsf.org
+[sav]: http://savannah.nongnu.org
 
 ## Support or Contact ##
 
 Please use the [GitHub issue tracker][ghtracker]. The [mailing list][mail] will also remain active for the time being.
 
 [ghtracker]: https://github.com/sol1/rdiff-backup/issues
 [mail]: https://lists.nongnu.org/mailman/listinfo/rdiff-backup-users]
-
-## Requirements: ##
```

Comment:
imagining we don't need all this info on the index, github readme seems like more suitable place?